### ISSUE TITLE
the stream is done or WriteHeader was already called

### ISSUE THREAD

**Issue by stanfieldr** [storj/storj#2608]
is this error concerning?
```
ERROR	piecestore protocol: rpc error: code = Internal desc = transport: transport: the stream is done or WriteHeader was already called
	storj.io/storj/storagenode/piecestore.(*Endpoint).Upload:328
	storj.io/storj/pkg/pb._Piecestore_Upload_Handler:701
	storj.io/storj/pkg/server.logOnErrorStreamInterceptor:23
	google.golang.org/grpc.(*Server).processStreamingRPC:1209
	google.golang.org/grpc.(*Server).handleStream:1282
	google.golang.org/grpc.(*Server).serveStreams.func1.1:717
```

**Comment by ifraixedes** [comment]
Thanks for reporting it.

We have identified this issue and we have a ticket in our issue tracker to tackle it (https://storjlabs.atlassian.net/browse/V3-1428).

**Comment by ifraixedes** [comment]
Yes, it isn't resolved, for this reason, the ticket is still open.

**Comment by stefanbenten** [comment]
this Issue should be resolved by the release since v0.25.0. I am going to close this, but please feel free to report back in case you still see this happening.
